.github/workflows/pr-open.yml

```diff
@@ -123,7 +123,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Deploys
-        uses: bcgov-nr/action-deployer-openshift@v1.0.0
+        uses: bcgov-nr/action-deployer-openshift@bug/templateFix
```

Comment:
Waiting for https://github.com/bcgov-nr/action-deployer-openshift/pull/16 to be merged and released before moving forward

Reply:
Done!